modules/platform/nuxeo-retention-core/src/main/resources/OSGI-INF/retention-schedulers.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.retention.schedulers">
+    
+  <extension target="org.nuxeo.ecm.platform.scheduler.core.service.SchedulerRegistryService" point="schedule">
+    <schedule id="findRetentionExpired">
+      <eventId>findRetentionExpired</eventId>
+      <!-- every hour -->
+      <cronExpression>0 0 * * * ?</cronExpression>
+    </schedule>
+  </extension>
+
+</component>
```

Comment:
I would add a newline here, for sanity and also to keep in sync with 10.10 which has this newline. Code should be identical in backports and master if at all possible.